src/query/frontend/ast/ast.lcp

```diff
@@ -2266,14 +2267,16 @@ cpp<#
     #>cpp
     AuthQuery(Action action, std::string user, std::string role,
               std::string user_or_role, Expression *password,
-              std::vector<Privilege> privileges, std::vector<std::string> labels)
+              std::vector<Privilege> privileges, std::vector<std::string> labels,
+              std::vector<std::string> edgeTypes)
         : action_(action),
           user_(user),
           role_(role),
           user_or_role_(user_or_role),
           password_(password),
           privileges_(privileges),
-          labels_(labels) {}
+          labels_(labels),
+          edge_types_(edgeTypes) {}
```

Comment:
We are using `sneak_case`-d variable names, thus `edgeTypes` should be `edge_types`.